xml/System.Threading.Tasks/Parallel.xml

```diff
@@ -950,6 +950,7 @@
   
  Any methods in the source orderable partitioner return <see langword="null" />.</exception>
         <related type="Article" href="https://docs.microsoft.com/previous-versions/msp-n-p/ff963552(v=pandp.10)">Parallel Loops</related>
```

Comment:
I prefer to leave the exceptions together before the links start. 

```suggestion
        <exception cref="T:System.AggregateException">The exception that is thrown to contain an exception thrown from one of the specified delegates.</exception>
        <related type="Article" href="https://docs.microsoft.com/previous-versions/msp-n-p/ff963552(v=pandp.10)">Parallel Loops</related>
```

Reply:
This exception description though doesn't help explain why you get that exception. @stephentoub 
would it be correct to say 

`<exception cref="T:System.AggregateException">One of the specified delegates has thrown an exception.</exception>`?

Reply:
Note: There were a bunch of AggregateExceptions with this same message.

Reply:
Yep, I saw that. So once we confirm the text, we can just search and replace.